src/core/Modal/Modal/Modal.tsx

```diff
@@ -11,31 +11,35 @@ import { ModalContentProps } from '../ModalContent/ModalContent';
 export type ModalVariant = 'smallScreen' | 'default';
 
 export interface ModalProps {
-  /** Show or hide the modal */
+  /** Shows or hides the modal */
   visible: boolean;
-  /** Application root element id for setting aria-hidden where needed */
+  /** Application root element id for setting aria-hidden appropriately */
```

Comment:
where needed/appropriately are both a bit vague, what to take into account when assigning id?

Reply:
This is now addressed in the latest commit